code.js

```diff
@@ -0,0 +1,115 @@
+const cost = 'Cur_OfficialRate';
+const abr = 'Cur_Abbreviation';
+const scale = 'Cur_Scale';
+const inp = 'in_';
+let hidden = true;
+const select = ' ✓';
+const con = 'con_';
+
+document.getElementById('currency_list').hidden = true;
+
+const exchangeRate = {
+	url: "https://www.nbrb.by/api/exrates/rates?periodicity=0",
+	currency: new Map(),
+	rate: ['USD', 'EUR', 'RUB', 'UAH'],
+	async getRate(){
+		let response = await fetch(exchangeRate.url);
+		let data = await response.json();
+		data.forEach((name, index) => {
+			exchangeRate.currency.set(name[abr], data[index]);
+		});
+		this.showRate();
+		this.fill();
+	},
+	showRate(){
+		this.rate.forEach((curName) => {
+			document.getElementById(curName).textContent = exchangeRate.currency.get(curName)[cost];
+			this.calculate(curName, 1);
+		});
+	},
+	fill(){
+		document.getElementById(inp + 'BYN').value = 1;
+	},
+	calculate(curName, sum){
+		let res = (Math.round(((sum / this.currency.get(curName)[cost] * this.currency.get(curName)[scale]))*100)/100);
+		document.getElementById(inp + curName).value = res;
+	},
+	update(obj){
+		let curName = obj.id.slice(3, 6);
```

Comment:
again, why let? don't see any of changes

Reply:
replaced with a constant